src/client/baseClient.ts

```diff
@@ -219,7 +288,8 @@ export class BaseClient {
     const query: BaseQuery = {
       v: invokeData.apiVersion,
       skillId: data?.skillId,
-      externalSystem: invokeData.externalSystem,
+      externalSystem:
```

Comment:
We should also set user-agent as faas-client-node

Reply:
If not done somewhere else

Reply:
this was already by default 
![image](https://github.com/user-attachments/assets/b58b1ab9-9e60-431c-8238-88d668d947fe)

Reply:
and is "faas-client-js"